src/styles/theme.scss

```diff
@@ -169,16 +188,28 @@ h5 {
   }
 }
 
+.link[href*="#/stories"] {
```

Comment:
Why has this been changed from the standard one?

Reply:
what do you mean? the href value?

Reply:
I mean the link colour - why this link needs it's own colour.